src/mono/mono/eventpipe/test/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ if(ENABLE_PERFTRACING)
         
         set(EPTEST_LINKABLE_LIBS "")
         list(APPEND EPTEST_LINKABLE_LIBS eglib_api monosgen-static ${OS_LIBS} ${LLVM_LIBS} monoapi)
-        if (CLR_CMAKE_USE_SYSTEM_ZLIB)
+        if (CLR_CMAKE_USE_SYSTEM_ZLIB OR CLR_CMAKE_HOST_ARCH_WASM)
```

Comment:
This extra condition is unnecessary here. CLR_CMAKE_USE_SYSTEM_ZLIB should include wasm.